### PR TITLE
Expose a simple validate address function in the API.

### PR DESCRIPTION
**Pull request by zoidbergZA**

### CODE REVIEW

**Comment by ExtraHash** [comment]
This is already present in ValidateParameters, is there some reason it needs to be in the WalletBackend  module as well?

https://turtlecoin.github.io/turtlecoin-wallet-backend-js/modules/_validateparameters_.html#validateaddresses

**Comment by zpalmtree** [comment]
I agree with extra. If we want to make a simple true/false that only takes one address, I can see that being worthwhile - but perhaps should be in ValidateParameters as well?

**Comment by zoidbergZA** [comment]
Ah yes you're right @ExtraHash. The idea was to have a simpler version of the function that can be used in form validation in the front-end for example, therefore the boolean return type. I will move it to the correct module

**Comment by ExtraHash** [comment]
a simple one address input version of the function would be handy for sure

File: lib/ValidateParameters.ts [hunk 1]
```diff
@@ -63,6 +63,26 @@ export function validateAddresses(
...
+export function validateAddress(
+    address: string,
+    integratedAddressAllowed: boolean,
+    config?: IConfig): boolean {
+
+    const err: WalletError = validateAddresses(
+        new Array(address), integratedAddressAllowed, MergeConfig(config),
+    );
+
+    return err.errorCode === 0;
```
Comment (zpalmtree) [inline lib/ValidateParameters.ts]:
```suggestion
    return err.errorCode === WalletErrorCode.SUCCESS;
```

**Comment by zpalmtree** [comment]
Thanks for the PR!

**Pull request merged**